scipy/stats/_relative_risk.py

```diff
@@ -0,0 +1,197 @@
+
+from collections import namedtuple
+import numpy as np
+from scipy.special import ndtri
+
+
+ConfidenceInterval = namedtuple('ConfidenceInterval', ['low', 'high'])
+
+
+class RelativeRiskResult:
+
+    def __init__(self, exposed_cases, exposed_total,
+                 control_cases, control_total,
+                 relative_risk):
+        self.relative_risk = relative_risk
+        self.exposed_cases = exposed_cases
+        self.exposed_total = exposed_total
+        self.control_cases = control_cases
+        self.control_total = control_total
```

Comment:
should these check `>0`? otherwise will be division by `0` in the `log(rr)` calculation, unless I missed something.

Reply:
Good point.  I'll add checks for 0 and handle it appropriately.

Reply:
Done.

Reply:
The latest commit is `f56eea7...`:
```
In [5]: relative_risk(1, 0, 1, 1)                                               
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-5-c4e8ffb0b537> in <module>
----> 1 relative_risk(1, 0, 1, 1)

~/mc37allpip/lib/python3.7/site-packages/scipy-1.6.0.dev0+f56eea7-py3.7-macosx-10.9-x86_64.egg/scipy/stats/_relative_risk.py in relative_risk(exposed_cases, exposed_total, control_cases, control_total)
    234 
    235     exposed_cases = _validate_int(exposed_cases, 0, "exposed_cases")
--> 236     exposed_total = _validate_int(exposed_total, 1, "exposed_total")
    237     control_cases = _validate_int(control_cases, 0, "control_cases")
    238     control_total = _validate_int(control_total, 1, "control_total")

~/mc37allpip/lib/python3.7/site-packages/scipy-1.6.0.dev0+f56eea7-py3.7-macosx-10.9-x86_64.egg/scipy/stats/_relative_risk.py in _validate_int(n, bound, name)
     13         raise TypeError(msg) from None
     14     if n < bound:
---> 15         raise ValueError(msg)
     16     return n
     17 

ValueError: exposed_total must be an integer not less than 1, but got 0
```